packages/components/src/Chip/Chip.css

```diff
@@ -88,8 +88,7 @@
   --chip--gradient-color-variation: var(--chip-base-bg);
 }
 
-.base:hover,
-.base:focus-visible {
+.base:hover {
```

Comment:
I've recently realized that Chip actually gets a hover whether or not it's interactive, which shouldn't be the case - possible to change that here?

Reply:
as in if there's no `onClick` it should not have a hover nor `cursor: pointer`?

we do have this in the AC

>Out of scope is introduction of any new state to chip or any new styling changes (will be covered in different ticket)

would you be alright with us addressing that separately?

Reply:
your comment did make me notice something that we'd want to address now though

when you have an onClick the height is 40. when you don't, and we render a `div` it is 42px tall.

<img width="340" alt="image" src="https://github.com/user-attachments/assets/56b6a415-118d-44d5-b685-75a2a48bbf06">

Reply:
ah. it's because `button` gets `box-sizing: border-box` from the user agent stylesheet and `div` does not so it ends up being `content-box` and adds the 2pixels of height to 40 rather than making it be 40 accounting for the border

Reply:
Rule 26 of front-end development: It's always box-sizing